README.md

```diff
@@ -71,115 +67,48 @@ The widget generator will include a **Gulpfile.js** and the necessary package.js
 
 Earlier versions of the widget generator added Grunt as the default taskrunner. We included this option as well if you want to use this.
 
-\* **We do not recommend using jQuery in your widgets, please [read this issue](https://github.com/mendix/AppStoreWidgetBoilerplate/issues/38). For simple DOM-manipulation you can use Dojo, which is provided by Mendix. Only use jQuery when you need it for certain external libraries that depend on it.**
-
 ---
 
-## Use a taskrunner for development
-
-The widget generator will scaffold a widget for you. It provides a convenient setup to develop you widget and publish it on Github. To make it even more easy, we add files for taskrunners.
+### 2.2. Which template do you want to use for the widget?
 
+#### BadgeWidgetBoilerplate
 
-### Gulp
+The badge boiler plate is a react fully developed mendix widget that Shows a value as a badge or a color label which is recomended for beginners. Its has the following features
 
-The following tasks are provided in the **Gulpfile.js** and can be started by running ``gulp <TASKNAME>`` or ``npm run <TASKNAME>``:
+* Display as a badge or a color label
+* Attach an onclick microflow and microflow
+* Set static data text when the dynamic data is not specified
 
-* ``default``
+#### Empty widget
 
-You can omit this taskname and just run ``gulp``. This will watch for any changes in your ``src`` directory. If a change occurs, it will automatically create a new **MPK** and copy this to your ``widgets`` directory.
+The empty template is a mendix react hello world widget recommended for more experienced developers.
 
-It will also copy any changed Javascript files to your deployment folder. This means you do not have to redeploy locally, you can just refresh your browser. For changes in ``XML`` or ``CSS`` you will still need to synchronize your project directory (press F4 in the modeler) and redeploy.
+Note: all boilerplate are designed based on typescript .tsx file extension.
+With .tsx files allow usae of all the JSX (JavaScript XML) code, JSX is an embeddable XML-like syntax. It is meant to be transformed into valid JavaScript. In order to use JSX you must do two things.
```

Comment:
- Correct spelling of usage
- Consider the re-write:  `It is transformed into valid JavaScript code. To use JSX, follow these steps:-`
- Correct to `.TSX files` for consistency